tailcfg/tailcfg.go

```diff
@@ -2262,7 +2262,14 @@ type ControlDialPlan struct {
 // connecting to the control server.
 type ControlIPCandidate struct {
 	// IP is the address to attempt connecting to.
-	IP netip.Addr
+	IP netip.Addr `json:",omitzero"`
+
+	// ACEHost, if non-empty, means that the client should connect to the
+	// control plane using an HTTPS CONNECT request to the provided hostname. If
+	// the IP field is also set, then the IP is the IP address of the ACEHost
+	// (and not the control plane) and DNS should not be used. The target (the
+	// argument to CONNECT) is always the control plane's hostname, not an IP.
+	ACEHost string `json:",omitempty"`
```

Comment:
Consistent and modern?
``` suggestion
	ACEHost string `json:",omitzero"`
```

Reply:
Oh. I only asked because you introduced `omitzero` in the field before:
https://github.com/tailscale/tailscale/blob/71c38d994e1e3b85f7f6f2f72d6208ffcc117b11/tailcfg/tailcfg.go#L2265-L2266

Obviously, that one can’t be `omitempty`, but that was the impetus.